OpenRA.Game/TraitTypeIndex.cs

```diff
@@ -0,0 +1,72 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
```

Comment:
make check

Checking for explicit interface violations...

Checking for code style violations in OpenRA.Game...
Could not load signature of StyleCop.StyleCopCore:SignalOutput due to: Could not load file or assembly or one of its dependencies. assembly:Microsoft.Build.Framework, Version=3.5.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a type:<unknown type> member:<none>
Could not load signature of StyleCop.StyleCopCore:SignalOutput due to: Could not load file or assembly or one of its dependencies. assembly:Microsoft.Build.Framework, Version=3.5.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a type:<unknown type> member:<none>
Error: Utility application crashed. See utility.log for details

Unhandled Exception:
System.TypeLoadException: Failure has occurred while loading a type.
  at StyleCop.AlertDialog.SendToOutput (StyleCop.StyleCopCore core, System.String message, MessageBoxIcon icon) <0x40d25f00 + 0x000c7> in <filename unknown>:0 
  at StyleCop.AlertDialog.Show (StyleCop.StyleCopCore core, System.Windows.Forms.Control parent, System.String message, System.String title, MessageBoxButtons buttons, MessageBoxIcon icon) <0x40d25df0 + 0x000a7> in <filename unknown>:0 
  at StyleCop.StyleCopCore.LoadAddins (System.String path, System.Byte[] publicKey) <0x40d21cb0 + 0x00747> in <filename unknown>:0 
  at StyleCop.StyleCopCore.Initialize (ICollection`1 addInPaths, Boolean loadFromDefaultPath) <0x40d213f0 + 0x00103> in <filename unknown>:0 
  at StyleCop.StyleCopConsole..ctor (System.String settings, Boolean writeResultsCache, System.String outputFile, ICollection`1 addInPaths, Boolean loadFromDefaultPath, System.Object hostTag) <0x40ce9880 + 0x0013b> in <filename unknown>:0 
  at StyleCop.StyleCopConsole..ctor (System.String settings, Boolean writeResultsCache, System.String outputFile, ICollection`1 addInPaths, Boolean loadFromDefaultPath) <0x40ce9820 + 0x00047> in <filename unknown>: